doc/deprecated.md

```diff
@@ -109,6 +109,22 @@ info, see [#7463] and [#5976].
 
 </details>
 
+<details>
+<summary>2024-12-31: retiring legacy Pubsub admin APIs</summary>
```

Comment:
The following is more consistent. We probably also want to alphabetize the library entries, like we do in the CHANGELOG:

```
<summary>Pubsub: remove legacy admin APIs</summary>
```

Reply:
done